wp-content/mu-plugins/pantheon/pantheon-page-cache.php

```diff
@@ -263,6 +266,17 @@ public function view_settings_page() {
 		<?php
 	}
 
+	/**
+	 * Set a stronger cache-control header for admin or logged in requests.
+	 *
+	 * This removes "max-age=0" which could hypothetically be used by
+	 * Varnish on an immediate subsequent request.
+	 *
+	 * @return void
+	 */
+	public function no_cache_add_headers() {
+		header( 'cache-control: no-cache, no-store, must-revalidate');
```

Comment:
It looks like there should be a corresponding space by the closing parenthesis (for symmetry).

Reply:
Updated